ssv/types.go

```diff
@@ -35,10 +36,12 @@ type AttesterCalls interface {
 
 // ProposerCalls interface has all block proposer duty specific calls
 type ProposerCalls interface {
+	// SubmitValidatorRegistration submits a validator registration
+	SubmitValidatorRegistration(pubkey []byte, feeRecipient bellatrix.ExecutionAddress, sig phase0.BLSSignature) error
 	// GetBeaconBlock returns beacon block by the given slot and committee index
 	GetBeaconBlock(slot phase0.Slot, committeeIndex phase0.CommitteeIndex, graffiti, randao []byte) (ssz.Marshaler, spec.DataVersion, error)
```

Comment:
Let's remove the `committeIndex` from GetBeaconBlock as well, because there isn't a committee for proposals

Reply:
done @moshe-blox